ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -727,17 +727,19 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
     }
 
     addressIsBlank(address: Address): boolean {
-        return !address || this.countOfFieldsWithData(address) === 0;
+        return !address || !this.countOfFieldsWithData(address);
     }
 
     enoughDataToSave(address: Address | null): boolean {
-        return address && this.countOfFieldsWithData(address) >= 1;
+        return address && this.countOfFieldsWithData(address);
     }
 
-    countOfFieldsWithData(address: Address): number {
-        const isNonBlankString: FuncType<boolean> = (val) => util.isString(val) && val.trim().length > 0;
-        const propsToCheck: (keyof Address)[] = ['name', 'country', 'street1', 'street2', 'state', 'city', 'zip', 'phone'];
-        return propsToCheck.map(prop => address[prop]).filter((value) => isNonBlankString(value)).length;
+    countOfFieldsWithData(address: Address): boolean {
```

Comment:
Name of the method does not fit the returned type.
`countOf..  => boolean`
It is a bit confusing.